include/dali/core/tensor_layout.h

```diff
@@ -302,10 +312,29 @@ class TensorLayout {
 
   DALI_HOST_DEV
   friend constexpr TensorLayout operator+(const TensorLayout &a, const TensorLayout &b);
+  DALI_HOST_DEV
+  friend constexpr TensorLayout operator+(const TensorLayout &a, const char &b);
```

Comment:
```suggestion
  friend constexpr TensorLayout operator+(const TensorLayout &a, const char b);
```

Reply:
Yeah, it is a leftover (I was trying to figure out a compile error, that turned out to be the lack of operator+=). I'll revert to pass by value